src/basic/embeds/position_embed.jl

```diff
@@ -18,9 +18,9 @@ get_value(e::PositionEmbedding, name::Symbol, xs::NamedTuple) = e(first(xs))
 
 function PE(size, pos, i::Int)
     if rem(i, 2) == 0
```

Comment:
```suggestion
    if rem(i, 2) == 1
```

Let's switch the order .